packages/webapp/src/containers/SensorReadings/saga.js

```diff
@@ -27,7 +27,9 @@ import {
 import { sensorUrl } from '../../apiConfig';
 import { getHeader } from '../../containers/saga';
 import { findCenter } from './utils';
-import { AMBIENT_TEMPERATURE, CURRENT_DATE_TIME } from './constants';
+import { CURRENT_DATE_TIME } from './constants';
+import { getTemperatureValue } from '../../components/Map/PreviewPopup/utils.js';
+import { TEMPERATURE } from './constants';
```

Comment:
Can we put these constants on line 21? Both `CURRENT_DATE_TIME` and `TEMPERATURE` are being imported from other constants on line 21

Reply:
Done